src/librustc/session/mod.rs

```diff
@@ -77,6 +77,10 @@ pub struct Session {
     /// available in this crate
     pub available_macros: RefCell<HashSet<Name>>,
 
+    /// Map from local spans for imported macros to the
```

Comment:
Could you be more explicit about what "local spans" are? And maybe a better name for the field is `imported_macro_spans`